apisix/utils/log-util.lua

```diff
@@ -94,6 +94,7 @@ local function get_full_log(ngx, conf)
 
     local log =  {
         request = {
+            id = var.request_id,
```

Comment:
Better not add a new field to the common library without full discussion.

Reply:
OK, deleted